app/build.gradle

```diff
@@ -137,7 +137,7 @@ def serverEdison = 'server';
 def serverEmulatorHost = '10.0.2.2';
 
 // Server on the Fantastica wifi network.
-def serverFantastica = '192.168.1.161';
+def serverFantastica = '192.168.1.179';
```

Comment:
Is this change something that should be checked in? I have no idea what this WiFi network is :)